src/explorer/WebAppTreeItem.ts

```diff
@@ -48,7 +51,16 @@ export class WebAppTreeItem extends SiteTreeItem {
         const tier: string = String(appServicePlan.sku!.tier);
         // tslint:disable-next-line:no-non-null-assertion
         this.deploymentSlotsNode = /^(basic|free|shared)$/i.test(tier) ? new DeploymentSlotsNATreeItem(tier, appServicePlan.id!) : new DeploymentSlotsTreeItem(this.client);
-        return [this.deploymentSlotsNode, this.folderNode, this.logFolderNode, this.webJobsNode, this.appSettingsNode];
+        const nodes = [this.deploymentSlotsNode, this.folderNode, this.logFolderNode, this.webJobsNode, this.appSettingsNode];
+        if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length === 1) {
+            const currentWorkspace = vscode.workspace.workspaceFolders[0];
+            const workspaceConfig = vscode.workspace.getConfiguration(constants.extensionPrefix, currentWorkspace.uri);
```

Comment:
You only need to pass in a uri to getConfiguration if the setting is unique per folder. For example, `defaultWebAppToDeploy` is unique for each project, so we listed the scope [here](https://github.com/Microsoft/vscode-azureappservice/blob/master/package.json#L650) as "resource".

This setting doesn't need to be unique per folder in my opinion and thus should have the default scope. Which means you don't need to pass in the uri